aspnetcore/grpc/aspnetcore.md

```diff
@@ -47,16 +47,75 @@ gRPC requires the [Grpc.AspNetCore](https://www.nuget.org/packages/Grpc.AspNetCo
 
 ### Configure gRPC
 
-gRPC is enabled with the `AddGrpc` method:
+In *Startup.cs*:
 
-[!code-csharp[](~/tutorials/grpc/grpc-start/sample/GrpcGreeter/Startup.cs?name=snippet&highlight=7)]
+* gRPC is enabled with the `AddGrpc` method.
+* Each gRPC service is added to the routing pipeline through the `MapGrpcService` method.
 
-Each gRPC service is added to the routing pipeline through the `MapGrpcService` method:
-
-[!code-csharp[](~/tutorials/grpc/grpc-start/sample/GrpcGreeter/Startup.cs?name=snippet&highlight=24)]
+[!code-csharp[](~/tutorials/grpc/grpc-start/sample/GrpcGreeter/Startup.cs?name=snippet&highlight=7,24)]
 
 ASP.NET Core middlewares and features share the routing pipeline, therefore an app can be configured to serve additional request handlers. The additional request handlers, such as MVC controllers, work in parallel with the configured gRPC services.
 
+### Configure Kestrel
+
+Kestrel gRPC endpoints:
+
+* Required HTTP/2.
+* Should be secured with HTTPS.
+
+#### HTTP/2
+
+Kestrel endpoints default to HTTP/1.1, and require additional configuration to enable HTTP/2. The *appsettings.json* defaults endpoints to the HTTP/2 connection protocol:
```

Comment:
Kestrels 3.0 default is HTTP/1.1 and HTTP/2.0. That config disables HTTP/1.1.

Reply:
This doc needs to be updated then: https://docs.microsoft.com/en-us/aspnet/core/fundamentals/servers/kestrel?view=aspnetcore-3.0&branch=pr-en-us-14015#http2-support

Reply:
I should be getting to the HTTP/2 update within a couple of weeks on https://github.com/aspnet/AspNetCore.Docs/issues/10141.